### DISCUSSION TITLE
Layouteinstellung wird nicht übernommen

### DISCUSSION THREAD

**Discussion by marcel511** [Zefau/ioBroker.jarvis#972] [General]
Die Anordnung und Größeneinstellung der Widget unter Layout wird auf einem 7" (800 x 480) Touchscreen nicht dargestellt. Alles ordnet sich selbständig und immer gleich an, egal was man im Layout einstellt.

![Layouteinstellung](https://user-images.githubusercontent.com/90475868/132904458-f0534a74-c59f-4197-a8b3-f28f51791bd8.PNG)
![Touchscreen](https://user-images.githubusercontent.com/90475868/132904460-717edaa0-85f3-4e67-9f35-037a718264f7.PNG)

**Comment by Zefau** [comment]
Nimm das Smartphone Layout

**Reply by marcel511** [reply]
Leider der gleiche Effekt, oder muss ich das bewusst irgendwo auswählen, dass das Smartphone Layout verwendet werden soll?

![Layouteinstellung2](https://user-images.githubusercontent.com/90475868/132906188-9f43731b-f4a4-4eaf-a169-2c119a40e5e8.PNG)

**Reply by Zefau** [reply]
Hmm ne, das ist blöd. Wird auf Tablet natürlich nicht angewandt. Da muss ich mir noch was ausdenken

**Reply by marcel511** [reply]
okay, vielen Dank schonmal.
